_source/_posts/2021-12-06-spring-data-elasticsearch.md

```diff
@@ -0,0 +1,369 @@
+---
+layout: blog_post
+title: "A Quick Guide to Elasticsearch with Spring Data and Spring Boot"
+author: jimena-garbarino
+by: contractor
+communities: [java]
+description: ""
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image:
+type: conversion
+---
+
+For sure you have heard about Elasticsearch or the Elastic Stack. It seems it started as a search engine based on Lucene, an open-source search engine library built by Shay Banon to index his wife's cooking recipes. From the early days, Elasticsearch has gone a long way and evolved into the Elastic Stack, a great suite for taking data from any source, search, analyze and visualize it in near real-time.
```

Comment:
```suggestion
You've probably heard of Elasticsearch or the Elastic Stack. The project started as a search engine based on [Lucene](https://lucene.apache.org/), an open-source search engine library built by Shay Banon to index his wife's cooking recipes. Since its early days, Elasticsearch has come a long way and has evolved into the [Elastic Stack](https://www.elastic.co/elastic-stack), a great suite for taking data from any source, in order to search, analyze, and visualize it in near real-time.
```
edits for grammar, style-guide, and idiomatic correctness, also added product links since we are no longer measured on sign-ups/leads